packages/apify/src/configuration.ts

```diff
@@ -16,6 +16,7 @@ export interface ConfigurationOptions extends CoreConfigurationOptions {
     actorRunId?: string;
     actorTaskId?: string;
     apiBaseUrl?: string;
+    apiPublicBaseUrl?: string;
```

Comment:
lets add a comment here about the difference between `apiBaseUrl` and `apiPublicBaseUrl`